File: snippets/csharp/ServiceStatus.cs
using Breez.Sdk;

public class WebhooksSnippets
{
    public void HealthCheckStatus(BlockingBreezServices sdk)
    {
        // ANCHOR: health-check-status
        try
        {
            var healthCheck = BreezSdkMethods.ServiceHealthCheck("<api key>");
            Console.WriteLine($"Current service status is: {healthCheck.status}");
        }
        catch (Exception)
        {
            // Handle error
        }
        // ANCHOR_END: health-check-status
    }

    public void ReportPaymentFailure(BlockingBreezServices sdk)
    {
        // ANCHOR: report-payment-failure
        var paymentHash = "...";
        try
        {
            sdk.ReportIssue(new ReportIssueRequest.PaymentFailure(
                new ReportPaymentFailureDetails(paymentHash)
            ));
        }
        catch (Exception)
        {
            // Handle error
        }
        // ANCHOR_END: report-payment-failure
    }
}

File: snippets/go/service_status.go
package example

import (
	"log"

	"github.com/breez/breez-sdk-go/breez_sdk"
)

func HealthCheckStatus() error {
	// ANCHOR: health-check-status
	healthCheck, err := breez_sdk.ServiceHealthCheck("<api key>")
	if err != nil {
		return err
	}
	log.Printf("Current service status is: %v", healthCheck.Status)
	// ANCHOR_END: health-check-status
	return nil
}

func ReportPaymentFailure() error {
	// ANCHOR: report-payment-failure
	paymentHash := "..."
	reportIssueRequest := breez_sdk.ReportIssueRequestPaymentFailure{
		Data: breez_sdk.ReportPaymentFailureDetails{
			PaymentHash: paymentHash,
		},
	}
	err := sdk.ReportIssue(reportIssueRequest)
	if err != nil {
		log.Printf("%#v", err)
		return err
	}
	// ANCHOR_END: report-payment-failure
	return nil
}
